source/localizable/tutorial/autocomplete-component.md

```diff
@@ -17,7 +17,7 @@ As before when we created the [`rental-listing` component](../simple-component),
 In our `app/templates/rentals.hbs` template file, we'll add a reference to our new `list-filter` component.
 
 Notice that below we "wrap" our rentals markup inside the open and closing mentions of `list-filter` on lines 12 and 20.
-This is an example of the [**block form**](/components/wrapping-content-in-a-component) of a component,
+This is an example of the [**block form**](/v2.12.0/components/wrapping-content-in-a-component) of a component,
```

Comment:
Thanks for the quick response locks, I've made the change. Just wondering, why do some use ../../ like on line 59?